chef/cookbooks/nova/recipes/trusted_flavors.rb

```diff
@@ -37,6 +37,10 @@
   if ssl_insecure
     novacmd = "#{novacmd} --insecure"
   end
+  if keystone_settings['api_version'] != "2.0"
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.